starters/express-typeorm-postgres/package.json

```diff
@@ -1,11 +1,10 @@
 {
-  "name": "express-typescript-typeorm-postgres",
+  "name": "express-typeorm-postgres",
   "version": "0.0.1",
   "description": "express, typescript, REST",
   "author": "This Dot Labs",
   "license": "MIT",
   "scripts": {
-    "postinstall": "docker-compose pull",
```

Comment:
Curious about this—not sure what it was for, so equally not sure why it needs to go.

Reply:
Yes, It is a remnant of a PR before this, that I forgot to remove before and I noticed it now. Basically, if the user does not have docker installed or running, this would fail their npm installs, and since in the previous PR we removed the "need" for pulling docker images by splitting up infrastructure and development mode API serving into two different commands we can remove this liability.